starknet_py/tests/e2e/account/account_test.py

```diff
@@ -884,3 +884,34 @@ async def test_account_execute_v3(account, deployed_balance_contract):
         call=get_balance_call
     )
     assert initial_balance + 100 == balance_after_increase
+
+
+@pytest.mark.asyncio
+async def test_execute_v3_with_get_chain_id(
```

Comment:
What is the rationale behind this test?

Reply:
It was the first test I added and the aim was to check if I could init Account with get_chain_id response. I've forgotten to remove it. thx for catch